Formula/linux-headers@5.15.rb

```diff
@@ -0,0 +1,21 @@
+class LinuxHeadersAT515 < Formula
+  desc "Header files of the Linux kernel"
+  homepage "https://kernel.org/"
+  url "https://cdn.kernel.org/pub/linux/kernel/v5.x/linux-5.15.57.tar.gz"
+  sha256 "d33d0e62f05d90d579eb0cc51343f48c32eb11e4fef0306c374b908212f51578"
+  license "GPL-2.0-only"
+
+  keg_only :versioned_formula
+
+  depends_on "rsync" => :build
+  depends_on :linux
+
+  def install
+    system "make", "headers_install", "INSTALL_HDR_PATH=#{prefix}"
+    rm Dir[prefix/"**/{.install,..install.cmd}"]
```

Comment:
```suggestion
    rm prefix.glob("**/{.install,..install.cmd}")
```
You're not operating on directories, so let's avoid `Dir`.